business/istio_status.go

```diff
@@ -237,7 +237,7 @@ func (iss *IstioStatusService) getAddonComponentStatus() IstioComponentStatus {
 
 	go getAddonStatus("prometheus", true, extServices.Prometheus.IsCore, &extServices.Prometheus.Auth, extServices.Prometheus.URL, extServices.Prometheus.HealthCheckUrl, staChan, &wg)
 	go getAddonStatus("grafana", extServices.Grafana.Enabled, extServices.Grafana.IsCore, &extServices.Grafana.Auth, extServices.Grafana.InClusterURL, extServices.Grafana.HealthCheckUrl, staChan, &wg)
-	go getAddonStatus("jaeger", extServices.Tracing.Enabled, extServices.Tracing.IsCore, &extServices.Tracing.Auth, extServices.Tracing.InClusterURL, extServices.Tracing.HealthCheckUrl, staChan, &wg)
```

Comment:
I guess that we should remove the `healthCheckUrl` because the URL used now will always be present. The URL now is static and is part of the Jaeger API. So I think it doesn't have any sense anymore now.

This config was set because of this request: https://github.com/kiali/kiali/issues/3631